dGame/dComponents/RocketLaunchLupComponent.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "Entity.h"
+#include "GameMessages.h"
+#include "Component.h"
+
+/**
+ * Component that handles the LUP/WBL rocket launchpad that can be interacted with to travel to WBL worlds.
+  *
+ */
+class RocketLaunchLupComponent : public Component {
+public:
+	static const uint32_t ComponentType = eReplicaComponentType::COMPONENT_TYPE_ROCKET_LAUNCH_LUP;
+
+	RocketLaunchLupComponent(Entity* parent);
+	~RocketLaunchLupComponent() override;
+
+	void OnUse(Entity* originator) override;
+
```

Comment:
Ideally let's have some comments in this header file for the methods and variables so future editors know what these are.